test-cases/longevity/longevity-large-partition-200k_pks-4days-tombstone-gc.yaml

```diff
@@ -0,0 +1,75 @@
+test_duration: 6480
```

Comment:
do you have a staging run of this job?

Reply:
https://github.com/scylladb/scylla-cluster-tests/pull/4755#issuecomment-1144734793

Reply:
i see both have failed